packages/yoga/src/Result/native/Tags.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { arrayOf, shape } from 'prop-types';
+import styled from 'styled-components';
+
+import Tag from '../../Tag';
+
+const TagStyled = styled(Tag.Informative)`
+  margin-right: 4px;
+`;
+
+const Wrapper = styled.ScrollView`
+  display: flex;
+  flex-direction: row;
+  ${({
+    theme: {
+      yoga: {
+        spacing: { xxsmall },
+      },
+    },
+  }) => `
+  margin-top: ${xxsmall}px;
+  `}
+`;
+
+const ResultTags = ({ items }) => {
+  return (
+    <Wrapper horizontal showsHorizontalScrollIndicator={false}>
+      {items &&
+        items.map(({ ...props }) => <TagStyled size="small" {...props} />)}
+    </Wrapper>
+  );
+};
```

Comment:
```suggestion
const ResultTags = ({ items }) => (
    <Wrapper horizontal showsHorizontalScrollIndicator={false}>
      {items.map(({ ...props }) => <TagStyled size="small" {...props} />)}
    </Wrapper>
  );
```

`items` will always be truthy in this component

Reply:
Done! @ggdaltoso